src/ray/gcs/gcs_client/service_based_gcs_client.cc

```diff
@@ -204,7 +204,7 @@ void ServiceBasedGcsClient::ReconnectGcsServer() {
       if (last_reconnect_address_ == address &&
           (current_sys_time_ms() - last_reconnect_timestamp_ms_) <
               RayConfig::instance().minimum_gcs_reconnect_interval_milliseconds()) {
-        RAY_LOG(INFO)
+        RAY_LOG(DEBUG)
```

Comment:
Maybe we keep this log level as INFO or even upgrade it to WARNING? It's useful when we try to  figure out what happened while client reconnecting. (although it prints much logs)

Reply:
Please avoid using WARNING logs unless the error message clearly indicates what to do to end users. All of the warning logs from GCS server and raylet will be streamed to drivers. (If we'd like to use WARNING more aggressively, we should change this behavior first).

Reply:
Got it. How about keeping it as INFO? @ffbin

Reply:
It prints much logs, so DEBUG is better. If client reconnect, we have print log `Reconnected to GCS server:`.